operator/autopilot-channels/packages/configconnector/1.125.0/crds.yaml

```diff
@@ -45782,18 +47500,12 @@ spec:
     storage: true
     subresources:
       status: {}
-status:
```

Comment:
Why remove this block?

Reply:
This is a good change from controller-gen. We don't want to deploy object (CRDs or other kind) to the cluster with a status.